crates/nu-command/src/bytes/to_str.rs

```diff
@@ -0,0 +1,118 @@
+use super::{operate, BytesArgument};
+use nu_engine::CallExt;
+use nu_protocol::ast::{Call, CellPath};
+use nu_protocol::engine::{Command, EngineState, Stack};
+use nu_protocol::{
+    Category, Example, PipelineData, ShellError, Signature, Span, SyntaxShape, Value,
+};
+
+struct Arguments {
+    column_paths: Option<Vec<CellPath>>,
+}
+
+impl BytesArgument for Arguments {
+    fn take_column_paths(&mut self) -> Option<Vec<CellPath>> {
+        self.column_paths.take()
+    }
+}
+
+#[derive(Clone)]
+pub struct BytesToStr;
+
+impl Command for BytesToStr {
+    fn name(&self) -> &str {
+        "bytes to-str"
```

Comment:
I'm sure that `to-str` was in my gist defining byte commands but now that I see it, it makes me wonder how it's different from `decode utf-8`? If there is some difference, or we intend to make it different somehow in the future, then we should keep it. If there is no difference, I'm not sure we should keep it.

Reply:
There is just a little difference, `to-str` only support to `utf-8`, and it support passing column path, so we can use `to-str` like this:
```
[[ColA ColB ColC]; [0x[61 73 64 66] 0x[67 68 69 66] 0x[71 77 65 72]]] | bytes to-str ColA ColC
```

But I think `decode` is more feature rich, I'm ok to remove it

Reply:
maybe we can add the column path passing to decode as well, and then we wouldn't need this?